### ISSUE TITLE
Fails for April 2021 Database

### ISSUE THREAD

**Issue by aneziac** [epogrebnyak/weo-reader#20]
Running the following code
```
from weo import download

download(year=2021, release="Apr", filename="weo.csv")
```
results in `curl` returning the HTML not found page instead of the csv.

The link for the April 2021 database is https://www.imf.org/-/media/Files/Publications/WEO/WEO-Database/2021/WEOApr2021all.xls. So, it seems likely that IMF went back to their original URL format and October 2020 is just an anomaly. Or maybe only October releases will now include the `/02/`. The code needs to be fixed either way.

**Comment by epogrebnyak** [comment]
@aneziac 

Does https://www.imf.org/-/media/Files/Publications/WEO/WEO-Database/2021/WEOApr2021all.ashx

point to:

https://www.imf.org/-/media/Files/Publications/WEO/WEO-Database/2021/WEOApr2021all.xls

?


so that .ashx just becomes .xls?

**Comment by epogrebnyak** [comment]
This saves a readable file:

```
curl -o weo.xls https://www.imf.org/-/media/Files/Publications/WEO/WEO-Database/2021/WEOApr2021all.ashx
```

Should we structure the links to this format?

**Comment by epogrebnyak** [comment]
```
import weo
weo.download(2021, 'Apr', 'weo-2021-Apr.xls')
```

seems to work now as a hotfix for just latest month, can you check?

**Comment by aneziac** [comment]
Yeah that hotfix works fine for me. I believe .ashx does point to the .xls files. I think that most likely the IMF will use the `https://www.imf.org/-/media/Files/Publications/WEO/WEO-Database/2021/WEOApr2021all.ashx` format, and that Oct 2020 is just a special case.

**Comment by epogrebnyak** [comment]
Thanks! We should also test the link  backwards to earlier years, will
submit some code (check file size without downloading)

сб, 17 апр. 2021 г., 3:07 Nate ***@***.***>:

**Comment by aneziac** [comment]
Looks like 2007 is still the limit unfortunately

**Comment by epogrebnyak** [comment]
Closed by 31e7842968b03363c54db5827f3626bb5dc015bc (temp fix become permanent)

**Comment by epogrebnyak** [comment]
> Looks like 2007 is still the limit unfortunately

Maybe https://github.com/datasets/imf-weo has some earlier data
